DDG4/CMakeLists.txt

```diff
@@ -113,10 +113,31 @@ IF(TARGET EDM4HEP::edm4hep)
 ENDIF()
 
 IF(DD4HEP_USE_HEPMC3)
+  set(DDG4HepMC3_INCLUDES $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/hepmc> $<BUILD_INTERFACE:${HEPMC3_INCLUDE_DIR}>)
+  set(DDG4HepMC3_USES DD4hep::DDG4 Geant4::Interface ${HEPMC3_LIBRARIES})
+  set(DDG4HepMC3_DEFINITIONS)
+
+  IF(ZLIB_FOUND)
+    list(APPEND DDG4HepMC3_INCLUDES ${ZLIB_INCLUDE_DIR})
+    list(APPEND DDG4HepMC3_USES ${ZLIB_LIBRARIES})
+    list(APPEND DDG4HepMC3_DEFINITIONS HEPMC3_USE_COMPRESSION HEPMC3_Z_SUPPORT=1)
+  ENDIF()
+  IF(LIBLZMA_FOUND)
+    list(APPEND DDG4HepMC3_INCLUDES ${LIBLZMA_INCLUDE_DIRS})
+    list(APPEND DDG4HepMC3_USES ${LIBLZMA_LIBRARIES})
```

Comment:
```suggestion
    list(APPEND DDG4HepMC3_USES LibLZMA::LibLZMA)
```
since cmake 3.14

Reply:
Would you mind bumping https://github.com/AIDASoft/DD4hep/blob/f6f3f7fc59fcec334aff7ee215c23287c11c54db/CMakeLists.txt#L1 to 3.14?

Reply:
Right, I need to check if we can.

Reply:
We bumped cmake to 3.14